src/h5web/visualizations/shared/AxisGrid.tsx

```diff
@@ -2,27 +2,27 @@ import React, { useState } from 'react';
 import { useThree, useFrame, Dom } from 'react-three-fiber';
 import { scaleLinear } from 'd3-scale';
 import IndexAxis from './IndexAxis';
-import styles from './HeatmapVis.module.css';
-import { useProps } from './hooks';
-import { Domain } from './models';
+import styles from './AxisGrid.module.css';
+import { Domain, AxisOffsets } from './models';
 
 interface AxisDomains {
   left: Domain;
   bottom: Domain;
 }
 
-function AxisGrid(): JSX.Element {
-  const { dims, axisOffsets } = useProps();
-  const [rows, cols] = dims;
+interface Props {
+  axisDomains: AxisDomains;
+  axisOffsets: AxisOffsets;
+}
```

Comment:
I feel the interface here should be consistent with the one of `Tooltip`. For that, I would change `left` in `y` and `bottom` in `x` and let `AxisGrid` the job of attributing `x` to `BottomAxis` and `y` to `LeftAxis`. What do you think ?

Reply:
Ah well, yep, agreed :D

Reply:
Ignoring my review comment, I would prefer having `xDomain` and `yDomain` as props like in `Tooltip`, and leave `AxisDomains` as is for use with `useState`. I feel that `x` and `y` could be mis-interpreted in some situations - e.g. `const {x, y} = axisDomains`.